lib/open_food_network/order_cycle_management_report.rb

```diff
@@ -130,6 +130,10 @@ def is_payment_methods?
       params[:report_type] == "payment_methods"
     end
 
+    def payments_total(payments)
+      payments_total = payments.sum(&:amount)
+    end
```

Comment:
Assignment not needed. Thanks CodeClimate :-)

Reply:
Ping devs - I have a question here.
This is broken as it is also showing Void payments and Incomplete payments.
I would be very grateful if someone could help me strip out Void and Incomplete payments such that the Payments total is accurate.
Thanks in advance!

Reply:
you shouldnt do biz logic here, the models do that for you.
Maybe you just need order.payment_total ?

Reply:
That is exactly the kind of wisdom I was looking for... Wisdom to help noobs :-)